charts/function-mesh-operator/values.yaml

```diff
@@ -59,3 +59,13 @@ controllerManager:
   # resourceLabels: {}
   # resource annotations applied to each function/connector managed by this controller
   # resourceAnnotations: {}
+
+  configFile: /etc/config/config.yaml
+  enable-leader-election: true
+  enable-pprof: false
+  metrics:
+    port: 8080
+  healthProbe:
+    port: 8000
+  pprof:
```

Comment:
how about move `enable-pprof` to `pprof` section, like:

```
pprof:
  enable: false
  port: 8090
```

Reply:
agree